src/main/java/com/teragrep/cfe_39/consumers/kafka/queue/QueueUtilities.java

```diff
@@ -0,0 +1,73 @@
+/*
+   HDFS Data Ingestion for PTH_06 use CFE-39
+   Copyright (C) 2022  Fail-Safe IT Solutions Oy
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+ */
+
+package com.teragrep.cfe_39.consumers.kafka.queue;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.function.BiPredicate;
+import java.util.function.ToLongFunction;
+
+class QueueUtilities {
```

Comment:
utility classes functionality should be moved to place it is used in

Reply:
Solved in commit 8ce5ca472e1852a2c7d7fa1f7a5374b396103858